content/consulting/net-8.mdx

```diff
@@ -0,0 +1,79 @@
+  project: .NET 8
+callToAction: 'Talk to us about your {{TITLE}} project today'
+benefits:
+  benefitList:
+    - image: /images/benefits/Inspection.png
+      title: 1. ASSESSMENT
+      description: |
+        Assessing your current solutions and their features
+        in a Specification Review.
+    - image: /images/benefits/ai.png
+      title: 2. STRATEGY
+      description: >
+        Gauging the appropriate strategy based on your specific needs and
+        priorities.
+      linkName: (Strangler Fig or Big Bang)
+      linkURL: 'https://www.ssw.com.au/rules/modernize-your-app/'
+    - image: /images/benefits/development-process.png
+      title: 3. PLAN
+      description: |
+        For each component, either:
+
+        Refactor - modernize the code to run with .NET 8
+
+        Replace - find a product with the same functionality
+
+        Rewrite – if there is no existing equivalent, rewrite from scratch
+    - image: /images/benefits/implementation.png
+      title: 4. MIGRATE
+      description: >
+        Executing the migration in a safe and controlled manner, minimizing
+        disruption and mitigating risk.
+    - image: /images/benefits/best-practices.png
+      title: 5. OPTIMIZE
+      description: >
+        Providing post-migration support and checking in to ensure your software
+        is performing as expected.
+technologies:
+  header: Related Technologies
+  technologyCards:
+    - technologyCard: content/technologies/dotnet.mdx
+    - technologyCard: content/technologies/cloud.mdx
+    - technologyCard: content/technologies/mobile-app-development.mdx
+    - technologyCard: content/technologies/web-applications.mdx
+    - technologyCard: content/technologies/ai.mdx
+    - technologyCard: content/technologies/ms-azure.mdx
+    - technologyCard: content/technologies/kubernetes.mdx
+    - technologyCard: content/technologies/Blazor.mdx
+    - technologyCard: content/technologies/api.mdx
+---
+
+# Optimize your application for the future.
+
+<VideoEmbed url="https://www.youtube.com/watch?v=zorbI0iDFQk&t=2s" />
+
+# Why .NET 8?
+
+.NET 8 is the latest LTS (Long Term Support) version of the platform – migrating now will futureproof your applications.
+
+.NET 8 provides robust enterprise functionality and cloud-native development all within a stable ecosystem backed by Microsoft.
+
+For your team, .NET 8 is the ultimate dev experience, updated with state-of-the-art features like C# 12, Native AOT (Ahead of Time Compilation), and source code generation.
+
+# Why SSW?
+
+Migration is complex, risky, and costly if not done correctly. SSW specializes in migrating applications, and our team of Microsoft certified consultants have a deep understanding of the .NET platform.
+
+We have an established history of helping clients migrate intricate systems to the newest version of .NET, and we have documented our learnings on our [Rules to Better .NET 8 Migration.](https://www.ssw.com.au/rules/rules-to-better-net8-migrations/)
```

Comment:
You can add this as the rule link as well if you want in the "rule" section of benefits:

![image](https://github.com/SSWConsulting/SSW.Website/assets/20507092/4d37e5dd-60d7-4b78-8245-cc986765513f)

![image](https://github.com/SSWConsulting/SSW.Website/assets/20507092/03e9f3d0-80e1-473b-b9b5-849d1c3a5cb2)